lib/lwm2m_carrier/Kconfig

```diff
@@ -33,12 +33,20 @@ menuconfig LWM2M_CARRIER
 
 if LWM2M_CARRIER
 
-config LWM2M_CARRIER_BOOTSTRAP_URI
-	string "LwM2M carrier bootstrap URI"
+config LWM2M_CARRIER_USE_CUSTOM_BOOTSTRAP_URI
+	bool "Use custom bootstrap URI"
+	help
+	  Use a custom bootstrap URI.
```

Comment:
I'd elaborate a bit in the help string that when this option is enabled, it's not only needed to specify the URI itself, but also provide the `bootstrap_psk.h` file.

Reply:
Agreed, will update